website/docs/registries/publishing.md

```diff
@@ -44,17 +46,35 @@ publish = false
 
 Use the `scarb package` command to create an archive of your package.
 You can read about the package compression algorithm and contents in the [Package tarball](./package-tarball) section.
-Basically when you run the command, Scarb gathers the source code of your package along with metadata files, such as the manifest file, and places them in an archive in `target/package` directory.
+Basically when you run the command, Scarb gathers the source code of your package along with metadata files, such as the
+manifest file, and places them in an archive in `target/package` directory.
 
-If you are in a Git repository, Scarb will first check if the repo state is clean and error out in case of any changes present in the Git working directory.
+If you are in a Git repository, Scarb will first check if the repo state is clean and error out in case of any changes
+present in the Git working directory.
 To bypass this check, you can use the `--allow-dirty` flag.
 
 The next step is package verification.
-After creating the initial archive, Scarb will attempt to unpack it and compile to check for any corruptions in the packaging process.
+After creating the initial archive, Scarb will attempt to unpack it and compile to check for any corruptions in the
+packaging process.
 If you want to speed up the packaging process, you can disable this step using the `--no-verify` flag.
 
 > [!WARNING]
 > This is a dangerous operation as it can lead to uploading a corrupted package to the registry.
 > Please use with caution.
 
-After successfully completing the whole process, the `{name}-{version}.tar.zst` archive waits in the `target/package` directory for being uploaded, where both `name` and `version` correspond to the values in `Scarb.toml`.
+After successfully completing the whole process, the `{name}-{version}.tar.zst` archive waits in the `target/package`
+directory for being uploaded, where both `name` and `version` correspond to the values in `Scarb.toml`.
+
+### Files included in the package
+
+All files in the package directory are included in the resulting tarball, except for the following:
+
+- Files excluded with rules defined in any `.scarbignore`, `.gitignore` or `.ignore` files.
+- The `.git` directory.
+- Any subdirectories containing `Scarb.toml` file.
+- The `<package root>/target` directory.
```

Comment:
nit:
`<package root>/target` is likely to be causing more problems so we can reorder these to simplify troubleshooting
```suggestion
- The `<package root>/target` directory.
- Any subdirectories containing `Scarb.toml` file.
- The `.git` directory.
```